README.md

```diff
@@ -86,6 +86,30 @@ user@host ~/neotron-pico-bios $ DEFMT_LOG=debug cargo run --release
 
 You should see your Neotron Pico boot, both over RTT in the `probe-run` output, and also on the VGA output.
 
+6a. Multiple probes
+
+If you have multiple probe-rs compatible probes attached to your computer,
+you will receive an error message.
+
+You can set the PROBE_RUN_PROBE environment variable to select one of the 
+available probes, like so:
+
+```console
+$ probe-run --list-probes
+the following probes were found:
+[0]: Picoprobe CMSIS-DAP (VID: 2e8a, PID: 000c, Serial: Exxxxxxxxxxxxxx6, CmsisDap)
+[1]: STLink V2 (VID: 0483, PID: 3748, Serial: 0xxxxxxxxxxxxxxxxxxxxxxE, StLink)
+user@host ~/neotron-pico-bios $ PROBE_RUN_PROBE='0483:3748' DEFMT_LOG=debug cargo run --release
+```
+
+You can also just provide the probe Serial, for example if you have multiple 
+identical probes.
+
+The documentation for this feature can be found at 
+https://github.com/knurling-rs/probe-run#12-multiple-probes 
+ -- 
```

Comment:
What does this line do?

Reply:
Umm, I remember wanting to add something, but after testing it didn't seem important.